components/notion/actions/update-page/update-page.mjs

```diff
@@ -5,7 +5,7 @@ export default {
   key: "notion-update-page",
   name: "Update Page",
   description: "Updates page property values for the specified page. Properties that are not set via the properties parameter will remain unchanged. [See the docs](https://developers.notion.com/reference/patch-page)",
-  version: "0.0.1",
+  version: "0.0.2",
```

Comment:
I think the version can be bumped to `0.1.0`, since [it adds functionality with a backwards compatible manner](https://pipedream.com/docs/components/guidelines/#versioning)

```suggestion
  version: "0.1.0",
```